src/operator/quantization/quantized_elemwise_mul.cc

```diff
@@ -247,9 +246,6 @@ NNVM_REGISTER_OP(_contrib_quantized_elemwise_mul)
 .add_arguments(QuantizeElemwiseMulParam::__FIELDS__());
 
 NNVM_REGISTER_OP(elemwise_mul)
-.set_attr<FQuantizable>("FQuantizable", [](const NodeAttrs& attrs) {
-    return QuantizeType::kMust;
```

Comment:
Keep this attribute as `kSupport`?

Reply:
default value is `kSupport`, so no need to add this.